tce/src/applibs/LLVMBackend/TDGen.cc

```diff
@@ -3473,6 +3473,9 @@ TDGen::writeInstrInfo(std::ostream& os) {
     // Call operations.
     writeControlFlowInstrDefs(os);
 
+    // Hardware loop instructions
+    writeHWLoopDef(os);
```

Comment:
Same here, it's adding them unconditionally now?

Reply:
I will add a flag here; I missed the case where we might generate IR separately and call backend with it.